CHANGELOG.md

```diff
@@ -22,6 +22,16 @@ Both the index and the values of the timeseries (both being date strings) should
 
 |changed| `inbuilt` math -> `pre-defined` math and `custom` math -> `pre-defined` math in the documentation.
 
+|changed| Calliope attribute dictionaries (AttrDicts) no longer sort dictionary keys on `union`. Key order is now: original dictionary key order + any new keys being added in the order they appear in the new dictionary.
+
+### Internal changes
+
+|new| `py.typed` file so that mypy recognises Calliope as a typed library when it is imported as a dependency.
+
+|changed| GitHub action versions updated: `codecov/codecov-action` -> v4, `actions/checkout` -> v4, `dawidd6/action-download-artifact` -> v3.
```

Comment:
Do we want CI changes in the changelog? We didn't in the past, but I realise some CI-related changes have been there since 0.7.0.dev2

Reply:
Maybe we don't, you're right